test/integ/test_py_version.py

```diff
@@ -13,34 +13,38 @@
 
 from __future__ import print_function
 
-from sagemaker import fw_utils
-from sagemaker.utils import sagemaker_timestamp
+import json
+import os
+
 import docker_utils
 import utils
-import os
-import json
+from sagemaker import fw_utils
+from sagemaker.utils import sagemaker_timestamp
 
 
 # The image should run the user-provided code using the right python version when training.
 def test_train_py_version(docker_image, sagemaker_session, py_version, opt_ml, processor):
     resource_path = 'test/resources/py_version/code'
 
     s3_source_archive = fw_utils.tar_and_upload_dir(session=sagemaker_session.boto_session,
-                                bucket=sagemaker_session.default_bucket(),
-                                s3_key_prefix=sagemaker_timestamp(),
-                                script='usermodule.py',
-                                directory=resource_path)
+                                                    bucket=sagemaker_session.default_bucket(),
+                                                    s3_key_prefix=sagemaker_timestamp(),
+                                                    script='usermodule.py',
+                                                    directory=resource_path)
 
     hp = _py_version_dict(py_version)
 
-    utils.create_config_files('usermodule.py', s3_source_archive.s3_prefix, opt_ml, additional_hp=hp)
+    utils.create_config_files('usermodule.py', s3_source_archive.s3_prefix, opt_ml,
+                              additional_hp=hp)
     os.makedirs(os.path.join(opt_ml, 'model'))
     docker_utils.train(docker_image, opt_ml, processor)
 
-    # The usermodule.py train_fn will assert on the expected python versions passed in through hyperparameters,
+    # The usermodule.py train_fn will assert on the expected
+    # python versions passed in through hyperparameters,
     # and training will fail if they are incorrect.
     success_file = 'output/success'
-    assert os.path.exists(os.path.join(opt_ml, success_file)), 'expected file not found: {}'.format(success_file)
+    assert os.path.exists(os.path.join(opt_ml, success_file)), 'expected file not found: {}'.format(
+        success_file)
```

Comment:
I think this one might be cleaner to either make `os.path.join(opt_ml, success_file)` its own variable or the error message rather than put `success_file)` on its own line